parser/prism/Translator.cc

```diff
@@ -30,13 +30,13 @@ std::unique_ptr<parser::Assign> Translator::translateAssignment(pm_node_t *untyp
     unique_ptr<parser::Node> lhs;
 
     if constexpr (std::is_same_v<PrismAssignmentNode, pm_constant_write_node>) {
-        auto nameLoc = &node->name_loc;
-        auto name = parser.resolveConstant(node->name);
-        lhs = make_unique<SorbetLHSNode>(parser.translateLocation(nameLoc), nullptr, gs.enterNameConstant(name));
+        // Handle regular assignment to a "plain" constant, like `A = 1`
+        lhs = translateConst<pm_constant_write_node, parser::ConstLhs>(node);
     } else if constexpr (std::is_same_v<PrismAssignmentNode, pm_constant_path_write_node>) {
-        auto isAssignment = true;
-        lhs = translateConstantPath(node->target, isAssignment);
```

Comment:
The migration goes:

`translateConstantPath(..., true) ` -> `translateConst<..., parser::ConstLhs>(...)`
`translateConstantPath(..., false)` -> `translateConst<..., parser::Const>(...)`